pinot-spi/src/main/java/org/apache/pinot/spi/config/table/assignment/InstanceAssignmentConfig.java

```diff
@@ -31,10 +31,12 @@
   @JsonPropertyDescription("Configuration for the instance tag and pool of the instance assignment (mandatory)")
   private final InstanceTagPoolConfig _tagPoolConfig;
 
-  @JsonPropertyDescription("Configuration for the instance constraints of the instance assignment, which filters out unqualified instances and sorts instances for picking priority")
+  @JsonPropertyDescription("Configuration for the instance constraints of the instance assignment,"
+      + " which filters out unqualified instances and sorts instances for picking priority")
   private final InstanceConstraintConfig _constraintConfig;
 
-  @JsonPropertyDescription("Configuration for the instance replica-group and partition of the instance assignment (mandatory)")
+  @JsonPropertyDescription(
+      "Configuration for the instance replica-group and partition of" + " the instance assignment (mandatory)")
```

Comment:
This looks slightly weird. One way to fix is to add empty comment `//` to the first line to force it to change the line